core/pubnub_pubsubapi.h

```diff
@@ -46,7 +46,7 @@ pubnub_t* pubnub_init(pubnub_t* p, const char* publish_key, const char* subscrib
     pubnub_done() on @p p, or the otherwise stop using it (like when
     the whole software/ firmware stops working). So, the contents of
     the @p uuid string is not copied to the Pubnub context @p p.  */
-void pubnub_set_uuid(pubnub_t* p, const char* uuid);
+enum pubnub_res pubnub_set_uuid(pubnub_t* p, const char* uuid);
```

Comment:
The comment above suggests that we do not copy the uuid string into the context. But with this change, we now do.

Reply:
@pnehrer Interesting comment, because it goes against what is inside of function. It was copying `uuid` string even in previous versions: https://github.com/pubnub/c-core/blob/master/core/pubnub_ccore_pubsub.c#L122

```c
void pbcc_set_uuid(struct pbcc_context* pb, const char* uuid)
{
    if (uuid != NULL) {
        PUBNUB_ASSERT_OPT(strlen(uuid) < sizeof pb->uuid);
        strncpy(pb->uuid, uuid, sizeof pb->uuid);
        pb->uuid[(sizeof pb->uuid) - 1] = '\0';
    }
    else {
        pb->uuid[0] = '\0';
    }
}
```